### PR TITLE
Fetch Rockets Data

### PR DESCRIPTION
**Pull request by momo-87**
### Added Features:
- Add rocketsSlice.js file and configure the store
- Fetch rockets data from the API
- Add extra reducer to rocketsSlice.js file
- Render data fetched from the API
- Update the README.md file

### CODE REVIEW

File: src/components/RocketsList.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { useSelector } from 'react-redux';
+import RocketItem from 'components/RocketItems';
+
+const RocketsList = () => {
+  const { rocketItems, isLoading, error } = useSelector((state) => state.rockets);
```
Comment (lordksix) [inline src/components/RocketsList.jsx]: Excellent use of useSelector to call the state of your rocketsSlice, however, I would suggest to use `(state) => state.rockets` within the slice so you do not repeat yourself everytime you need to call it

File: README.md [hunk 2]
```diff
@@ -170,6 +170,7 @@ Go to [🚀 Live Demo](https://bookstore-react-redux-kappa.vercel.app/) to check
 👤 **Author**
 
 - GitHub: [@lordksix](https://github.com/lordksix)
+- GitHub: [@momo-8*](https://github.com/momo-87)
```
Comment (lordksix) [inline README.md]: Maybe there is a typo in your GitHub username?

File: src/features/rockets/rocketsSlice.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  error: undefined,
+};
+
+const rocketsSlice = createSlice({
+  name: 'rockets',
+  initialState,
+  reducers: {},
+  extraReducers(builder) {
+    builder
+      .addCase(fetchRockets.pending, (state) => {
```
Comment (lordksix) [inline src/features/rockets/rocketsSlice.js:26]: Very good separation of promises. My recommendation would be to reset the error value in every case a change in status is made. Remember, the first time there might be an issue fetching the data, if you try a second time, your slice is going to remember the previous error value store in your slice

**Review by lordksix — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️ 

Awesome! 👏
 You did a great job setting up the rocket slice. 👌 
However, there are still some changes that need to be done before moving to the next step. 
Almost there


# Highlights ⭐ 
- Correct fetching of data👍 
- No linter error 👍 
- Descriptive commits 👍 

# Required changes ♻️ 
_Check the comments under the review_

# Optional Suggestions 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding  👨‍💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/RocketsList.jsx (hunk 1 above)
Comment (momo-87) [reply src/components/RocketsList.jsx]: Thank you for your suggestion. 🙏 
The required change has been made.

File: src/features/rockets/rocketsSlice.js (hunk 3 above)
Comment (momo-87) [reply src/features/rockets/rocketsSlice.js:26]: Awesome, thank you. The required change has been made.

File: README.md (hunk 2 above)
Comment (momo-87) [reply README.md]: Thank you for your observation, there was a mistake. The required change has been made.

**Review by lordksix — APPROVED** [review]
# Approved! 👍 
Congratulations!

**Pull request closed without merging**